common/app/components/Nav/Avatar-Points-Nav-Item.jsx

```diff
@@ -4,20 +4,25 @@ import { Link } from 'react-router';
 // this is separated out to prevent react bootstrap's
 // NavBar from injecting unknown props to the li component
 export default function AvatarPointsNavItem({ picture, points, username }) {
+  const placeholderImg = 'https://github.com/FreeCodeCamp/assets/blob/master' +
```

Comment:
Import this instead of re-defining it.

https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/common/utils/constantStrings.json#L2